compiler/rustc_expand/src/mbe/transcribe.rs

```diff
@@ -448,6 +448,12 @@ fn count_repetitions<'a>(
                 }
             }
             MatchedSeq(named_matches) => {
+                if named_matches.is_empty() {
+                    return Err(cx.struct_span_err(
+                        sp.entire(),
+                        "related fragment that refers the `count` meta-variable expression was not found",
+                    ));
```

Comment:
You could add a structured diagnostic for this new error

Reply:
Done